frame/election-provider-multi-phase/src/lib.rs

```diff
@@ -381,11 +381,11 @@ impl Default for ElectionCompute {
 #[derive(PartialEq, Eq, Clone, Encode, Decode, RuntimeDebug)]
 pub struct RawSolution<C> {
 	/// Compact election edges.
-	compact: C,
+	pub compact: C,
```

Comment:
It's not clear why these items need to be public.

Reply:
Useful for testing and building the miner in the future. 

They should either be private (which I am not a fan of for this pallet), or have proper functions to work with, at least like `::new`. A public struct with no methods and all-private fields is just nonsense and serves no purpose.